dispatcher/registry.py

```diff
@@ -82,11 +83,16 @@ def apply_async(self, args=None, kwargs=None, queue=None, uuid=None, connection=
 
         obj = self.get_async_body(args=args, kwargs=kwargs, uuid=uuid, **kw)
 
+        from dispatcher.conf import settings
+
+        for broker_name, broker_kwargs in settings.brokers.items():
```

Comment:
why we would have multiple brokers and why we would take just the first one? I admit I'm lost at this point.

Reply:
This was addressed by adding to the config. My thought process was still evolving when I first pushed this. Now this part should be coherent, although I can't yet get rid of the in-line import.

https://github.com/ansible/dispatcher/blob/a77cbb2aae302e32558a104aed0155bd94945943/dispatcher.yml#L22-L23